Leanplum-SDK/Classes/Managers/Networking/LeanplumRequest.m

```diff
@@ -87,16 +87,22 @@ - (id)initWithHttpMethod:(NSString *)httpMethod
 
 + (NSDictionary *)createHeaders {
     NSDictionary *infoDict = [[NSBundle mainBundle] infoDictionary];
-    NSString *userAgentString = [NSString stringWithFormat:@"%@/%@/%@/%@/%@/%@/%@/%@",
+    NSString *userAgentString = [NSString stringWithFormat:@"%@/%@/%@/%@/%@/%@/%@/%@/%@",
```

Comment:
why do we have `createHeaders` here and also in LPRequstSender.

Reply:
I had discussed this yesterday. LeanplumRequest.m (old request class , which is used currently) and LPRequstSender (new Request class, not used currently but might be used in future).

Reply:
While debugging through the code, i have noticed another issue with the way old and new request are incorrectly structured. Will discuss with you.

Reply:
LPRequestSender is used for new Requests. So, we needed there. I have made utils reusable method.